conda/meta.yaml

```diff
@@ -13,7 +13,7 @@ build:
 test:
   requires:
     - ts-conda-build =0.4
-    - ts-xml
+    - ts-xm5
```

Comment:
```suggestion
    - ts-xml
```

Reply:
No, I modified the wrong line. Fixed now.